.ci/dockerfiles/Dockerfile.gpu-test

```
@@ -41,9 +43,11 @@ RUN . /etc/os-release && \
         'Trusted: yes' | sudo tee /etc/apt/sources.list.d/ubuntu.sources >/dev/null && \
     grep -rlE 'archive\.ubuntu\.com|security\.ubuntu\.com|ports\.ubuntu\.com' /etc/apt/sources.list /etc/apt/sources.list.d/ 2>/dev/null | sudo xargs -r rm -f && \
     sudo apt-get update && \
-    sudo apt-get install -y --no-install-recommends gdb gdb-multiarch && \
-    .gitlab/build.sh ${NIXL_INSTALL_DIR} && \
-    sudo apt-get clean && \
+    sudo apt-get install -y --no-install-recommends gdb gdb-multiarch
+RUN --mount=type=secret,id=ssh_key,target=/tmp/ssh_key,mode=0400,uid=${_UID},gid=${_GID} \
```

Hadolint (2.15.1)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)